src/kern/arm/bsp/s32k5/Kconfig

```
@@ -0,0 +1,18 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: MIT
+
+# PF: S32K5
+# PFDESCR: NXP S32K5
+# PFSELECT: CAN_ARM_CPU_CORTEX_R52 ARM_GIC HAVE_ARM_GICV3 HAS_PLAT_AMP_OPTION
+# PFDEPENDS: ARM
+
+config PF_S32K5_AUTO_RAM_BASE
+	bool "Automatically choose RAM_BASE"
+	default y
+	help
+	  Use default RAM base address, CPE SRAM0.
+
+config PF_S32K5_MANUAL_RAM_BASE
+	hex "Custom RAM_BASE"
+	depends on !PF_S32K5_AUTO_RAM_BASE
+	default 0x22000000
```

As there is only a single cluster, we could simplify this IMHO:


```suggestion
config PF_S32K5_RAM_BASE
	hex "Kernel load address"
	default 0x22000000
	help
	  Start address of the kernel image. Defaults to start of CPE SRAM0.
```